src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,101 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  openedTodo: Todo | null;
+  setOpenedTodo: (todo: Todo) => void;
+  filter: string;
+  search: string;
+  standartList: Todo[];
+}
+export const TodoList: React.FC<Props> = ({
+  openedTodo,
+  setOpenedTodo,
+  filter,
+  search,
+  standartList,
+}) => {
+  const [list, setList] = useState<Todo[]>(standartList);
```

Comment:
repeating the same state in different components is not good. The solution to this problem is to pass a function that will change the state in the parent component